query-engine/core/src/error.rs

```diff
@@ -41,6 +41,14 @@ pub enum CoreError {
 
     #[error("{}", _0)]
     TransactionError(#[from] TransactionError),
+
+    #[error(
+        "Error converting field \"{}\" of expected type \"{}\", found incorrect value of \"{}\"",
+        _0,
+        _1,
+        _2
+    )]
+    FiedConversionError(String, String, String),
```

Comment:
Typo

Reply:
Style thoughts: I tend to do the above as well from time to time, but after some people slapped me on the wrist for it I agree with their view more: Tuple structs with more than one of the same type should be pulled into a non-tuple struct with proper names.